templates/assets/js/navigation.js

```diff
@@ -130,20 +129,46 @@ app.navigation = (function () {
                                 break;
                             case 'chart':
                                 app.chart.toggleChart();
+                                self.displayAllScenarios(scenarios);
                                 break;
                             case 'steps':
-                                self.displayAllScenarios(scenarios);
-                                self.toggleSteps(steps);
+                                if (document.querySelector('.error_btn').classList.contains('active')) {
```

Comment:
podbilem jsona. i dodalem aktualizacje currentView bo wywalalo to wysietlanie wszystkich stepow.